src/Fantomas.Core/CodePrinter.fs

```diff
@@ -1878,9 +1893,8 @@ let genTupleMultiline (node: ExprTupleNode) =
         function
         | Choice1Of2 e ->
             match e with
-            | Expr.IfThen _
-            | Expr.IfThenElif _
-            | Expr.IfThenElse _ when (idx < lastIndex) -> autoParenthesisIfExpressionExceedsPageWidth (genExpr e)
+            | _ when isSynExprIfThenElse e && (idx < lastIndex) ->
```

Comment:
Should be a partial active pattern. It is ok to have the when clause for `when (idx < lastIndex)`.